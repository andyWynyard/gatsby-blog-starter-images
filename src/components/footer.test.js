import React from "react";
import renderer from "react-test-renderer";
import Footer from "./footer";

describe("Footer", () => {
  it("Renders Correctly", () => {
    const data = {
      site: {
        siteMetadata: {
          homepage: "https://maverickbysigma.se/",
          people: "https://maverickbysigma.se/people/",
          work: "https://maverickbysigma.se/work/"
        }
      }
    };

    const { homepage, people, work } = data.site.siteMetadata;

    const tree = renderer
      .create(<Footer work={work} homepage={homepage} people={people} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
