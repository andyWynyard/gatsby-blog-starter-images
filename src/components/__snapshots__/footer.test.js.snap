// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Renders Correctly 1`] = `
<div
  className="footer"
>
  <div
    className="footer__content"
  >
    <p
      className="footer__question"
    >
      Want to get to know us better?
    </p>
    <p
      className="footer__question--two"
    >
      Interested in working on projects together?
    </p>
    <p
      className="footer__question--two"
    >
      Or interested in our previous work?
    </p>
    <p
      className="footer__links-title"
    >
      Check us out here
    </p>
    <p
      className="footer__links"
    >
      <a
        className="footer__link"
        href="https://maverickbysigma.se/work/"
      >
        Projects
      </a>
        
      /
        
      <a
        className="footer__link"
        href="https://maverickbysigma.se/"
      >
        Maverick Homepage
      </a>
        
      /
        
      <a
        className="footer__link"
        href="https://maverickbysigma.se/people/"
      >
        Our People
      </a>
    </p>
  </div>
</div>
`;
